File: src/BukkitPE/event/entity/EntityEvent.php
<?php
/**
 * Entity related Events, like spawn, inventory, attack...
 */
namespace BukkitPE\event\entity;

use BukkitPE\event\Event;

abstract class EntityEvent extends Event{
	/** @var \BukkitPE\entity\Entity */
	protected $entity;

	public function getEntity(){
		return $this->entity;
	}
}

File: src/net/BukkitPE/event/entity/EntityEvent.java
package net.BukkitPE.event.entity;

import net.BukkitPE.entity.Entity;
import net.BukkitPE.event.Event;

/**
 * author: MagicDroidX
 * BukkitPE Project
 */
public abstract class EntityEvent extends Event {

    protected Entity entity;

    public Entity getEntity() {
        return entity;
    }
}
